landcover.mss

```diff
@@ -20,8 +20,8 @@
 @commercial-line: #d1b2b0;  // Lch(75,12,25)
 @industrial: #ebdbe8;       // Lch(89,9,330) (Also used for railway)
 @industrial-line: #c6b3c3;  // Lch(75,11,330) (Also used for railway-line)
-@farmland: #fbecd7;         // Lch(94,12,80)
-@farmland-line: #d6c4ab;    // Lch(80,15,80)
+@farmland: #eef0d5;         // Lch(94,14,112)
+@farmland-line: #c7c9ae;    // Lch(80,14,112)
```

Comment:
We generally go for a bit more chroma on the lines.

Reply:
What's your proposition then?

Reply:
What do you think about https://github.com/gravitystorm/openstreetmap-carto/pull/3327#issuecomment-414098780? I can't see the difference.